specification/ai/Azure.AI.Projects/connections/models.tsp

```diff
@@ -27,15 +33,85 @@ model Connection {
   @visibility(Lifecycle.Read)
   target: string;
 
-  @doc("The authentication type used by the connection")
+  @doc("Whether the connection is tagged as the default connection of its type")
   @visibility(Lifecycle.Read)
-  authType: AuthenticationType;
+  isDefault: boolean;
 
   @doc("Metadata of the connection")
   @visibility(Lifecycle.Read)
   metadata: Record<string>;
 }
 
+@doc("A connection with API key authentication")
+model ConnectionWithApiKeyAuth extends Connection {
```

Comment:
do we need these strong typed models? Can't we just have base connection with optional credentials?